### PR TITLE
Lazy loading for responses

### PR DESCRIPTION
**Pull request by ManInFez**
Increase lazy loading for responses to prevent fetching schemas before its needed

### CODE REVIEW

File: ertviz/ert_client/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 from ertviz.models import Response, EnsembleModel
 
 
-def get_response(ref_url):
-    return Response(ref_url=ref_url)
+def get_response(name, ref_url):
```
Comment (xjules) [inline ertviz/ert_client/__init__.py:4]: Is this function even used somewhere?

Comment (ManInFez) [reply ertviz/ert_client/__init__.py:4]: Maybe not

Comment (xjules) [reply ertviz/ert_client/__init__.py:4]: Then we can remove it (from tests too I guess).

Comment (ManInFez) [reply ertviz/ert_client/__init__.py]: done!

File: ertviz/models/response.py [hunk 2]
```diff
@@ -4,32 +4,38 @@
...
+
+    @property
+    def ensemble_id(self):
+        if not self._schema:
+            self._schema = get_schema(api_url=self._ref_url)
+        return self._schema["ensemble_id"]
 
     @property
     def axis(self):
+        if not self._schema:
```
Comment (xjules) [inline ertviz/models/response.py:26]: this's quite repetitive, could we make a function e.g. `_update_schema` 
```python
def _update_schema
   if not self._schema:
          self._schema = get_schema(api_url=self._ref_url)
```

Comment (ManInFez) [reply ertviz/models/response.py]: :+1:

**Review by xjules — APPROVED** [review]
I think it is good to go now! Bra job!

**Pull request merged**